data/state/journalEntries_test.go

```diff
@@ -10,6 +10,184 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func TestNewJournalEntryCode_InvalidCodeHashShouldErr(t *testing.T) {
+	t.Parallel()
+
+	entry, err := state.NewJournalEntryCode([]byte{}, &mock.TrieStub{})
+	assert.Nil(t, entry)
+	assert.Equal(t, state.ErrInvalidHash, err)
+}
+
+func TestNewJournalEntryCode_NilUpdaterShouldErr(t *testing.T) {
+	t.Parallel()
+
+	entry, err := state.NewJournalEntryCode([]byte("code hash"), nil)
+	assert.Nil(t, entry)
+	assert.Equal(t, state.ErrNilUpdater, err)
+}
+
+func TestNewJournalEntryCode_OkParams(t *testing.T) {
+	t.Parallel()
+
+	entry, err := state.NewJournalEntryCode([]byte("code hash"), &mock.TrieStub{})
+	assert.Nil(t, err)
+	assert.NotNil(t, entry)
+}
+
+func TestJournalEntryCode_RevertErr(t *testing.T) {
+	t.Parallel()
+
+	updateErr := errors.New("update error")
+	codeHash := []byte("code hash")
+	ts := &mock.TrieStub{
+		UpdateCalled: func(key, value []byte) error {
+			return updateErr
+		},
+	}
+	entry, _ := state.NewJournalEntryCode(codeHash, ts)
+
+	acc, err := entry.Revert()
+	assert.Equal(t, updateErr, err)
+	assert.Nil(t, acc)
+}
+
+func TestJournalEntryCode_RevertUpdatesTheTrie(t *testing.T) {
+	t.Parallel()
+
+	updateCalled := false
+	codeHash := []byte("code hash")
+	ts := &mock.TrieStub{
+		UpdateCalled: func(key, value []byte) error {
+			assert.Equal(t, codeHash, key)
+			assert.Nil(t, value)
+			updateCalled = true
+			return nil
+		},
+	}
+	entry, _ := state.NewJournalEntryCode(codeHash, ts)
+
+	acc, err := entry.Revert()
+	assert.Nil(t, err)
+	assert.Nil(t, acc)
+	assert.True(t, updateCalled)
+}
+
+func TestJournalEntryCode_IsInterfaceNil(t *testing.T) {
+	t.Parallel()
+
+	entry, _ := state.NewJournalEntryCode([]byte("code hash"), &mock.TrieStub{})
+	assert.False(t, check.IfNil(entry))
+	entry = nil
+	assert.True(t, check.IfNil(entry))
+}
+
+func TestNewJournalEntryAccount_NilAccountShouldErr(t *testing.T) {
+	t.Parallel()
+
+	entry, err := state.NewJournalEntryAccount(nil)
+	assert.Nil(t, entry)
+	assert.Equal(t, state.ErrNilAccountHandler, err)
+}
+
+func TestNewJournalEntryAccount_OkParams(t *testing.T) {
+	t.Parallel()
+
+	entry, err := state.NewJournalEntryAccount(&mock.AccountWrapMock{})
+	assert.Nil(t, err)
+	assert.NotNil(t, entry)
```

Comment:
related to the comment above, use `assert.False(t, check.IfNil(entry))` here.
these 2 comments should be applied in the entire PR

Reply:
Done.